libs/features/mas/web-components/src/variants/mini-compare-chart.js

```diff
@@ -46,6 +46,9 @@ export class MiniCompareChart extends VariantLayout {
         'promo-text',
         'callout-content',
     ];
+    if (this.card.classList.contains('checkmark')) {
```

Comment:
Looking for a class added in Milo inside the web component code might seem a bit odd. I would suggest to move this code to Milo or move the footer row code to web component class. Ideally they should be decoupled.

thoughts @yesil, @npeltier @3ch023 ?

Reply:
Instead of using checkmark in the classlist, can I use it in the first row instead, similar to the hr color, chevron default open/close and footer-rows title?

Reply:
I can go with adjusting the footer rows height in milo if others agree. Only drawback I see is height alignment spread across 2 places.
How about changing the footer rows slot name for new mini compare design. Then I just need to add the new slot name here @Axelcureno ?

Reply:
On a second thought, I think it's okay to leave it as is. Anyways this card will only be used in Milo.